flink-table/flink-table-common/src/test/java/org/apache/flink/table/catalog/CatalogTestUtil.java

```diff
@@ -63,7 +79,22 @@ public static void checkEquals(CatalogTable t1, CatalogTable t2) {
 
     public static void checkEquals(CatalogView v1, CatalogView v2) {
         assertThat(v2.getTableKind()).isEqualTo(v1.getTableKind());
-        assertThat(v1.getSchema()).isEqualTo(v1.getSchema());
+
+        ResolvedSchema schema1;
+        if (v1 instanceof ResolvedCatalogView) {
```

Comment:
Can we extract it to a common method?
```
ResolvedSchema getResolvedSchema(CatalogBaseTable catalogBaseTable) {
        if (catalogBaseTable instanceof ResolvedCatalogBaseTable) {
            return ((ResolvedCatalogBaseTable<?>) catalogBaseTable).getResolvedSchema();
        } else {
            return resolve(catalogBaseTable.getUnresolvedSchema());
        }
    }
```
Also add some comment like: `we unify it to ResolvedSchema to compare` while calling this method.

Reply:
Good suggestion 👍🏻. Updated